pkg/noderesourcetopology/filter_test.go

```diff
@@ -141,6 +141,30 @@ func TestNodeResourceTopology(t *testing.T) {
 				},
 			},
 		},
+		&topologyv1alpha1.NodeResourceTopology{
```

Comment:
let's elaborate in the commit message the cases we want to cover. `scope=container` is a more complex scenario wrt to `scope=pod` (the latter is basically like the former with len(containers)=1) so we can make sure we have good coverage